sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.models.ts

```diff
@@ -11,3 +11,8 @@ export interface ScholasticStanding {
   dateOfWithdrawal?: string;
   scholasticStandingChangeType: StudentScholasticStandingChangeType;
 }
+
+/** Represents the scholastic standing summary details. */
```

Comment:
According to @guru-aot's comment it should be like:
<pre>
/**
* Comment
* /
</pre>